### PR TITLE
Update ja-JP.txt

### PR DESCRIPTION
**Pull request by Mitchykins**
Hopefully this came out right, and hopefully I'm doing this right for now on. Incase I accidentally do this wrong, I apologize.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
You did this correctly. :)
It can now be discussed.

**Comment by Gymnasiast** [comment]
@AaronVanGeffen

File: data/language/ja-JP.txt [hunk 1]
```diff
@@ -1369,14 +1369,14 @@ STR_1363    :Too high for supports!
...
+STR_1367    :半 ループ
+STR_1368    :半 口抜き (左)
+STR_1369    :半 口抜き (右)
 STR_1370    :Barrel Roll (left)
 STR_1371    :Barrel Roll (right)
 STR_1372    :Launched Lift Hill
-STR_1373    :Large Half Loop (left)
-STR_1374    :Large Half Loop (right)
+STR_1373    :大 半 ループ (左)
+STR_1374    :大 半 ループ (右)
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: Similar to above, I'd suggest to use 大きなハーフ・ループ here.

File: data/language/ja-JP.txt [hunk 2]
```diff
@@ -1385,7 +1385,7 @@ STR_1379    :Reverser (left)
 STR_1380    :Reverser (right)
 STR_1381    :Curved Lift Hill (left)
 STR_1382    :Curved Lift Hill (right)
-STR_1383    :Quarter Loop
+STR_1383    :四半期 ループ
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: Suggest we use "4分の1ループ"

File: data/language/ja-JP.txt [hunk 3]
```diff
@@ -1665,12 +1665,12 @@ STR_1658    :{WINDOW_COLOUR_2}intensity: {BLACK}less than {COMMA16}
 STR_1659    :{WINDOW_COLOUR_2}intensity: {BLACK}between {COMMA16} and {COMMA16}
 STR_1660    :{WINDOW_COLOUR_2}intensity: {BLACK}more than {COMMA16}
 STR_1661    :{WINDOW_COLOUR_2}Nausea tolerance: {BLACK}{STRINGID}
-STR_1662    :{WINDOW_COLOUR_2}Happiness:
-STR_1663    :{WINDOW_COLOUR_2}Nausea:
-STR_1664    :{WINDOW_COLOUR_2}Energy:
-STR_1665    :{WINDOW_COLOUR_2}Hunger:
-STR_1666    :{WINDOW_COLOUR_2}Thirst:
-STR_1667    :{WINDOW_COLOUR_2}Bathroom:
+STR_1662    :{WINDOW_COLOUR_2}幸福:
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: These are all rather literal. I'd suggest we use the terms from [this page](http://www39.atwiki.jp/rct3jpinfo/pages/36.html) instead.

File: data/language/ja-JP.txt [hunk 4]
```diff
@@ -1470,13 +1470,13 @@ STR_1464    :Helix up (small)
 STR_1465    :Helix up (large)
 STR_1466    :Helix down (small)
 STR_1467    :Helix down (large)
-STR_1468    :Staff
+STR_1468    :職員
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: Again, ironically, the RCT3 fan translation seems to favour the use of the katakana "スタッフ".

**Comment by Mitchykins** [comment]
Thanks for the comments

**Comment by AaronVanGeffen** [comment]
@Mitchykins Any update on this?

**Comment by Gymnasiast** [comment]
Closing this due to the lack of progress.

**Pull request closed without merging**